roles/wordpress-stack-install.yml

```diff
@@ -1,10 +1,8 @@
 ---
 - name: Install WordPress, MariaDB, Nginx, and PHP-FPM
   hosts: wordpress-server
-  remote_user: root
-  # remote_user: user
-  # sudo: yes
-
+  become: yes
```

Comment:
These two lines seem off to me...

Reply:
@jwflory Got it. You don't need `become_user` and it should be `become: true` in the latest versions of Ansible

Reply:
Which two lines? It actually is correct to use both `become` and `become_user`